vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/ExperimentInformationMain.java

```diff
@@ -138,20 +136,18 @@ private ExperimentId activeExperiment(ProjectId parsedProjectId) {
 
   private void setContext(Context context) {
     experimentContentComponent.setContext(context);
-    experimentSupportComponent.projectId(context.projectId().orElseThrow());
-    experimentSupportComponent.setExperiments(
-        experimentInformationService.findAllForProject(context.projectId().orElseThrow()));
+    experimentSupportComponent.setContext(context);
```

Comment:
What happened here?

Reply:
I introduced the Context object holding the context information in the ExperimentInformationMain and propagate it to the supportcomponent instead of the experimentID and projectID explicitly